integration_test/validate_metadata_test.go

```diff
@@ -18,7 +20,42 @@ var agentMetricsMetadata []byte
 var thirdPartyDataDir embed.FS
 
 func TestValidateMetadataOfThirdPartyApps(t *testing.T) {
```

Comment:
`TestThirdPartyPublicUrls` seems to be a super set of `TestValidateMetadataOfThirdPartyApps`. Do we choose to keep them separate so that it's easier to tell what type of problem it is when things fail? aka the test failure will clearly say it's because `TestThirdPartyPublicUrls` fails.

Reply:
Yes, that is the case.

Reply:
Thanks for confirming.